src/main/java/com/gengo/client/GengoClient.java

```diff
@@ -205,10 +214,57 @@ public JSONObject approveTranslationJob(int id, Rating rating,
         }
     }
 
+    /**
+     * Approve a translation.
+     * @param id The job ID
+     * @param rating A rating for the translation
```

Comment:
Describe that it rates both quality and response.

BTW, is it really correct behavior?  Should rating_quality and rating_time be null here?